dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventExporterTest.java

```diff
@@ -108,9 +112,18 @@ class EventExporterTest extends TrackerTest
 
     private Program program;
 
-    final Function<EventSearchParams, List<String>> eventsFunction = ( params ) -> eventService.getEvents( params )
-        .getEvents()
-        .stream().map( Event::getUid ).collect( Collectors.toList() );
+    final Function<EventOperationParams, List<String>> eventsFunction = ( params ) -> {
+        try
+        {
+            return eventService.getEvents( params )
```

Comment:
Maybe it is not part of this PR but this try/catch here seems dangerous.
We should re-evaluate this test when we tackle the service part.
Maybe add a comment here just saying to double check this later.